app/services/open_library_service.rb

```diff
@@ -0,0 +1,32 @@
+class OpenLibraryService
+  include HTTParty
+  base_uri 'https://openlibrary.org'
+
+  def initialize(format = 'json', jscmd = 'data')
+    @options = { query: { format: format, jscmd: jscmd } }
+  end
+
+  def book_info(identifier, identifier_type = 'ISBN')
+    @options[:query][:bibkeys] = "#{identifier_type}:#{identifier}"
```

Comment:
I added the identifier type optional parameter to allow searching books using other types of identifiers